setup.py

```diff
@@ -19,7 +19,10 @@
         "docker",
         "drmaa",
         "loguru",
+        "numpy",
```

Comment:
numpy was required already (for a spark thing iirc?) that failed the first time i tried to run linker. I added pandas and pyarrow.

Reply:
I don't understand why you need np. I don't have it in my env and things seem to work.

Reply:
https://github.com/ihmeuw/linker/pull/43/files
in `spark_utils`
but it also seems like the `diag` import isn't used at all--maybe better to just delete that instead.

Reply:
OK I did that